src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -1499,7 +1540,7 @@ describe('Decimal', () => {
 
     it('has the correct automation selectors', () => {
       render();
-      expect(wrapper.current.childAt(0).getDOMNode().getAttribute('data-component')).toBe('decimal');
+      expect(wrapper.current.find(Textbox).prop('data-component')).toBe('decimal');
```

Comment:
Previously this was making an assertion on the DOM (which is what we need for automation). Now, this only check's that the Textbox component has the prop, not that the Textbox component rendered into the DOM.

Could we revert the change please?